routing/bicycle_directions.cpp

```diff
@@ -385,7 +385,7 @@ void BicycleDirectionsEngine::FillPathSegmentsAndAdjacentEdgesMap(
     // It leads to preventing pushing item to |prevSegments|. So if there's no enough items in |prevSegments|
     // |pathSegment.m_trafficSegs| should be empty.
     // Note. For the time being BicycleDirectionsEngine is used for turn generation for bicycle and car routes.
-    if (m_generateTrafficSegs && prevSegments.size() + 1 == prevJunctionSize)
+    if (prevSegments.size() + 1 == prevJunctionSize)
```

Comment:
> Am I right that this is always true now?

Right. Let's do check instead. In next PR I'm writing:
```CHECK_EQUAL(prevSegments.size() + 1, prevJunctionSize, ());```

> Am I right that prevSegments.size() is always 1 at this point?

It's not so. The this is road point between joints is gathered in |prevJunctions| and segments between them is gathered in |prevSegments|. Please have a look at the lines:
```
    if (!IsJoint(ingoingEdges, outgoingEdges, inEdge, routeEdges[i], isCurrJunctionFinish,
                 inFeatureId.IsValid()))
    {
      continue;
    }
```
above in the method.

The comment above is not actual now and will be rewritten in next PR.
The field name |m_trafficSegs| is not actual now and will be renamed in next PR.